modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc

```diff
@@ -3,104 +3,73 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="attaching-additional-labels-to-your-time-series-and-alerts_{context}"]
-= Attaching additional labels to your time series and alerts
+
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+
+// tag::CPM[]
+[id="attaching-additional-labels-to-your-time-series-and-alerts-cpm_{context}"]
+= Attaching additional labels to your time series and alerts for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="attaching-additional-labels-to-your-time-series-and-alerts-uwm_{context}"]
+= Attaching additional labels to your time series and alerts for monitoring of user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples
+
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:component: prometheusK8s
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:component: prometheus
+// end::UWM[]
 
 You can attach custom labels to all time series and alerts leaving Prometheus by using the external labels feature of Prometheus.
```

Comment:
Suggestion: You can attach custom labels to all time series and alerts leaving Prometheus by using its external labels feature.

Reply:
I think addin "its" here can lead to double meaning. It would not be clear to me right away as a user if the feature is a label feature or Prometheus feature.